website/src/App.js

```diff
@@ -0,0 +1,40 @@
+import pic from "./PC_Group.jpg";
+import './App.css';
+import Navbar from "./components/Navbar/Navbar";
+
+
+function App() {
+  const header = ["Hi there!", <br />,"My name is Grace. Welcome to my website!"];
+  const about = ["I am a Dartmouth '24 from Houston, Texas.",<br />, "Currently, I am staying in Park City, Utah with a few friends."];
+  const funFacts = <ul>
+    <li>I LOVE food</li>
```

Comment:
Very small ui suggestion here - but since your are centering a list, the bullet points seem a *bit* out of alignment. Instead you could for-go the list, or try to come up with a fun new way to show the list items!